sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -1789,9 +1843,17 @@ pi_result piPlatformsGet(pi_uint32 NumEntries, pi_platform *Platforms,
     std::copy_n(PiPlatformsCache->begin(), NumEntries, Platforms);
   }
 
-  if (NumPlatforms)
+  if (NumPlatforms) {
     *NumPlatforms = PiPlatformsCache->size();
 
+    // Check if import user ptr into USM feature has been requested.
+    if (USMHostPtrImport) {
+      // Initialize availability of USM hostptr import feature.
+      // Since L0 is not platform-specific, just check the first Platform.
+      ZeUSMImport.setZeUSMImport(PiPlatformsCache->front());
+    }
```

Comment:
Please move to _pi_platform::initialize()  where "standard" driver's extensions are initialized.